glue/flagging_script_glue/flagging.py

```diff
@@ -797,13 +800,13 @@ def outlier_type(df: pd.DataFrame, condos: bool) -> pd.DataFrame:
             "Home flip sale (high)",
             "Family sale (high)",
             "Non-person sale (high)",
-            "Anomaly (high)",
+            # "Anomaly (high)",
```

Comment:
**[Question, non-blocking]** Do we want `Anomaly (high)` to continue to be commented out?

Reply:
Nope, thanks for catching.